content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -328,24 +322,24 @@ parameter.
 
 ### Access Modes
 
-A `PersistentVolume` can be mounted on a host in any way supported by the resource provider. As shown in the table below, providers will have different capabilities and each PV's access modes are set to the specific modes supported by that particular volume.  For example, NFS can support multiple read/write clients, but a specific NFS PV might be exported on the server as read-only. Each PV gets its own set of access modes describing that specific PV's capabilities.
+A PersistentVolume can be mounted on a host in any way supported by the resource provider. As shown in the table below, providers will have different capabilities and each PV's access modes are set to the specific modes supported by that particular volume.  For example, NFS can support multiple read/write clients, but a specific NFS PV might be exported on the server as read-only. Each PV gets its own set of access modes describing that specific PV's capabilities.
 
 The access modes are:
 
-* ReadWriteOnce -- the volume can be mounted as read-write by a single node
+* `Once -- the volume can be mounted as read-write by a single node
```

Comment:
Looks like these are nits introduced by a global replace operation. Please change it back.